### PR TITLE
Ic03 ic04 generate ce estab individual addressed print files

### PR DESCRIPTION
**Pull request by ryangrundy7**
# Motivation and Context

A test has been added for CE Estab initial contact Letters. CE Estab individual initial contact letters are different as they count all the expected responses and append rows to the printfile for them.
# What has changed

- Added a test for CE Estab initial contact Letters

# How to test?

- Run with other PRs on the ticket and run the tests, all tests should pass.

# Links

[Trello](https://trello.com/c/0WUGJq8e/)

### CODE REVIEW

**Comment by ryangrundy7** [comment]
[(IC03/IC04) Generate CE ESTAB Individual Addressed Print Files (21)](https://trello.com/c/0WUGJq8e/471-ic03-ic04-generate-ce-estab-individual-addressed-print-files-21)

File: resources/sample_files/sample_3_english_CE_estab.csv [hunk 1]
```diff
@@ -0,0 +1,4 @@
+UPRN,ESTAB_UPRN,ADDRESS_TYPE,ESTAB_TYPE,ADDRESS_LEVEL,ABP_CODE,ORGANISATION_NAME,ADDRESS_LINE1,ADDRESS_LINE2,ADDRESS_LINE3,TOWN_NAME,POSTCODE,LATITUDE,LONGITUDE,OA,LSOA,MSOA,LAD,REGION,HTC_WILLINGNESS,HTC_DIGITAL,TREATMENT_CODE,FIELDCOORDINATOR_ID,FIELDOFFICER_ID,CE_EXPECTED_CAPACITY,CE_SECURE,PRINT_BATCH
+123456789123,46488965919,CE,Prison,E,RD06,,Flat 56 Francombe House,Commercial Road,Tree area,Windleybury,XX1 0XX,51.4463421,-2.5924477,E00073438,E01014540,E02003043,E06000023,E12000009,1,5,CE_LFNR1E,1000010,100001001,5,0,37
+123456789124,46488965929,CE,Prison,E,RD06,,Flat 56 Francombe House,Commercial Road,Tree area,Windleybury,XX1 0XX,51.4463421,-2.5924477,E00073438,E01014540,E02003043,E06000023,E12000009,1,5,CE_LFNR1E,1000010,100001001,10,0,37
```
Comment (NickGrantX) [inline resources/sample_files/sample_3_english_CE_estab.csv]: could we make the addresses a bit different, just to make the test data a bit more realistic?

File: resources/sample_files/sample_3_welsh_CE_estab.csv [hunk 2]
```diff
@@ -0,0 +1,4 @@
+UPRN,ESTAB_UPRN,ADDRESS_TYPE,ESTAB_TYPE,ADDRESS_LEVEL,ABP_CODE,ORGANISATION_NAME,ADDRESS_LINE1,ADDRESS_LINE2,ADDRESS_LINE3,TOWN_NAME,POSTCODE,LATITUDE,LONGITUDE,OA,LSOA,MSOA,LAD,REGION,HTC_WILLINGNESS,HTC_DIGITAL,TREATMENT_CODE,FIELDCOORDINATOR_ID,FIELDOFFICER_ID,CE_EXPECTED_CAPACITY,CE_SECURE,PRINT_BATCH
+123456789123,79483231479,CE,Prison,E,RD06,,Flat 56 Francombe House,Commercial Road,Tree area,Windleybury,XX1 0XX,51.4463421,-2.5924477,W00073438,W01014540,W02003043,W06000023,W12000009,1,5,CE_LFNR1W,,,5,0,81
+123456789125,79483231473,CE,Prison,E,RD06,,Flat 56 Francombe House,Commercial Road,Tree area,Windleybury,XX1 0XX,51.4463421,-2.5924477,W00073438,W01014540,W02003043,W06000023,W12000009,1,5,CE_LFNR1W,,,5,0,81
```
Comment (NickGrantX) [inline resources/sample_files/sample_3_welsh_CE_estab.csv]: could we make the addresses a bit different, just to make the test data a bit more realistic?

**Review by NickGrantX — CHANGES_REQUESTED** [review]
Minor comment

**Review by NickGrantX — APPROVED** [review]
LGTM 👍

File: acceptance_tests/features/print_file.feature [hunk 3]
```diff
@@ -20,6 +20,21 @@ Feature: Scheduled print and manifest files can be generated and uploaded
...
       | D_ICA_ICLR2B   | CE_IC04_1   | [22]                | sample_1_welsh_CE_unit.csv           |
 
+  Scenario Outline: Generate print files and log events for initial contact letters CE Estabs
+    Given sample file "<sample file>" is loaded
+    And messages are emitted to RH and Action Scheduler with <questionnaire type> questionnaire types
+    When set action rule of type "<action type>" when the case loading queues are drained
+    And CE Estab messages are emitted to RH and Action Scheduler with <individual qid type> questionnaire types
+    Then correctly formatted "<pack code>" print files are created for CE Estab expected responses
+    And there is a correct "<pack code>" manifest file for each csv file written
+    And expected number of uac events logged against the case are [PRINT_CASE_SELECTED,SAMPLE_LOADED]
```
Comment (AdamHawtin) [inline acceptance_tests/features/print_file.feature]: Not sure about the wording of this step, what is a `uac event` and it says expected number but provides the actual list of events.

**Review by AdamHawtin — CHANGES_REQUESTED** [review]
Passes against the branches 👍 
Just a comment about step wording.

File: acceptance_tests/features/print_file.feature (hunk 3 above)
Comment (ryangrundy7) [reply acceptance_tests/features/print_file.feature]: Yeah it's a bit confusing, I'll try and reword it.

**Review by AdamHawtin — APPROVED** [review]
👍 Passed locally against the PR branches

**Pull request merged**